docs/reference/commandline/image_history.md

```diff
@@ -76,3 +77,43 @@ $ docker history --format "{{.ID}}: {{.CreatedSince}}" busybox
 f6e427c148a7: 4 weeks ago
 <missing>: 4 weeks ago
 ```
+
+### <a name="platform"></a> Show history for a specific platform (--platform)
+
+The `--platform` option allows you to specify which platform-variant to show
+history for if multiple platforms are present. By default, `docker history`
+shows the history for the daemon's native platform or if not present, the
+first available platform.
+
+If the local image store has multiple platform-variants of an image, the
+`--platform` option selects which variant to show the history for. An error
+is produced if the given platform is not present in the local image cache.
+
+The platform option takes the `os[/arch[/variant]]` format, for example,
+`linux/amd64` or `limux/arm64/v8`. Architecture and variant are optional,
+and default to the daemon's native architecture if omitted.
+
+The following examples pulls an `alpine` image for the daemon's native platform,
+and shows the history, then provides a custom platform, which fails until the
+given platform-variant is pulled;
```

Comment:
nit:
```suggestion
The following example: 
- pulls an `alpine` image for the daemon's native platform (because `--platform` is omitted);
- shows the history (also for the default platform, because `--platform` is omitted);
- tries to show the history for the image while providing a custom `--platform` value, which fails until the given platform-variant is pulled;
```

Reply:
Oh! Forgot to reply on these; I've been going back and forth a bit if this approach would be making things better (much more in depth) or if it would be "too much".

Happy to hear from others though. Also happy to make changes along these lines in a follow up.